Added SECURITY.md about responsible disclosure (#4)
vippsas/.github: SECURITY.md

# Security Policy

## TL;DR Responsible Disclosure

> If you find a security vulnerability in any of systems (including our mobile apps, web, APIs, or other systems we offer), please report it to us at on our [responsible disclosure page](https://vdp.vippsmobilepay.com).

Reviewer: ```suggestion
If you find a security vulnerability in any of our systems (including our mobile apps, web pages, APIs, or other systems we offer), please report it via our [responsible disclosure page](https://vdp.vippsmobilepay.com).
```

> We will appreciate that you give as much information as possible about the vulnerability and, if possible, a proof of concept and information on how to mitigate the vulnerability.

Reviewer: ```suggestion
We appreciate that you give as much information as possible about the vulnerability and, if possible, a proof of concept and information on how to mitigate the vulnerability.
```
